### PR TITLE
[#11] [UI] [Android] As a user, I can see the home screen after login success

### LINKED ISSUE #11: [UI] [Android] As a user, I can see the home screen after login success
## Why

After the user login success, navigate to the home screen as the landing page.

## Acceptance criteria

Show the home screen with:

- Current date information on the top left corner.
- `Today` title under the current date information.
- User profile image on the top right corner.

## Design

- [Design for the home screen](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=44%3A148&scaling=scale-down)

### PR DESCRIPTION
**Pull request by nmint8m**
- Close #11

## What happened 👀

In this PR, I built the based UI for the home screen on Android.

## Insight 📝

The following things won't be included in this PR:

- The font style for Android will not be implemented in this PR. It will be done by @thiennguyen0196 later.
- The navigation from the Splash screen to the Home screen will be done later.

## Proof Of Work 📹

image

### CODE REVIEW

File: android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/ui_models/HomeHeaderUiModel.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package co.nimblehq.kaylabruce.kmmic.android.presentation.ui_models
+
+class HomeHeaderUiModel(
```
Comment (markgravity) [inline android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/ui_models/HomeHeaderUiModel.kt:3]: What do you think about using the [data class](https://kotlinlang.org/docs/data-classes.html) here?

File: ios/ios/Sources/Constants/Dimens.swift [hunk 2]
```diff
@@ -10,7 +10,7 @@ import Foundation
 
 struct Dimens {
 
-    /// 4.0
+    
     static let tiny: CGFloat = 4.0
```
Comment (markgravity) [inline ios/ios/Sources/Constants/Dimens.swift:14]: Missing the `/// 4.0` here 🙏

**Review by markgravity — APPROVED** [review]

File: android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/constants/Dimens.kt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+object Dimens {
+    /// 4.0
+    const val tiny = 4.0
+    /// 10.0
+    const val small = 10.0
+    /// 16.0
+    const val medium = 16.0
+    /// 20.0
+    const val large = 20.0
+}
```
Comment (thiennguyen0196) [inline android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/constants/Dimens.kt:12]: Missing end of file here @nmint8m 🙏 You can enable this auto setting in Android Studio so you won't bother about this issue in the future.

Screenshot 2023-08-11 at 6 03 23 PM

File: android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/modules/home/HomeScreen.kt [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            }
+        }
+    }
+}
+
+@Preview
+@Composable
+fun PreviewHomeScreen() {
+    HomeScreen()
+}
```
Comment (thiennguyen0196) [inline android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/modules/home/HomeScreen.kt:46]: Ditto here

File: android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/modules/home/home_header/HomeHeaderView.kt [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+                model = uiModel.imageUrl,
+                contentDescription = null,
+                modifier = Modifier
+                    .size(Dp(36F))
+                    .clip(CircleShape)
+                    .clickable { onProfileImageTap() },
+            )
+        }
+    }
+}
```
Comment (thiennguyen0196) [inline android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/modules/home/home_header/HomeHeaderView.kt:52]: Ditto here

File: android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/ui_models/HomeHeaderUiModel.kt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package co.nimblehq.kaylabruce.kmmic.android.presentation.ui_models
+
+data class HomeHeaderUiModel(
+    val imageUrl: String,
+    val dateText: String,
+    val todayText: String,
+)
```
Comment (thiennguyen0196) [inline android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/ui_models/HomeHeaderUiModel.kt:7]: Ditto here

**Review by thiennguyen0196 — COMMENTED** [review]
Missing EOF in some files, the rest LGTM

**Review by thiennguyen0196 — APPROVED** [review]
LGTM now

**Pull request merged**